packages/aws-cdk/lib/cli/cdk-toolkit.ts

```diff
@@ -85,6 +85,13 @@ export interface CdkToolkitProps {
    */
   ioHost?: CliIoHost;
 
+  /**
+   * Name of the toolkit stack to use/deploy
+   *
+   * @default CDKToolkit
+   */
+  toolkitStackName?: string;
```

Comment:
@kaizencc This is the new option on the toolkit itself. Already exists on the new toolkit.